crates/cheatcodes/src/utils.rs

```diff
@@ -247,6 +285,38 @@ pub(super) fn sign_with_wallet<DB: DatabaseExt>(
     Ok(encode_vrs(sig))
 }
 
+pub(super) fn sign_eip2098_with_wallet<DB: DatabaseExt>(
```

Comment:
Essentially the only difference between the two functions is how the signature is encoded. I would prefer we return the signature object from `sign_with_wallet` and call the corresponding encode function in the cheatcode `impl` blocks

Reply:
done!